io-engine/src/bdev/nvmx/controller.rs

```diff
@@ -86,12 +102,22 @@ impl<'a> NvmeControllerInner<'a> {
             Some(NvmeControllerIoChannel::destroy),
         ));
 
-        let adminq_poller = poller::Builder::new()
+        // Select next non-master core for adminq poller.
```

Comment:
Is this selecting the non-master core, or non-current core?

Reply:
It supposed that controller creation code is executed on the master (gRPC) core.

Reply:
I will fix comment.